flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/history/ArchivedJson.java

```diff
@@ -55,6 +57,13 @@ public boolean equals(Object obj) {
 		}
 	}
 
+	public int hashCode() {
```

Comment:
Just use Objects.hash here. And it needs the override annotation.

Reply:
ok. I'll change this and push another commit. Thanks for the review @haohui